### ISSUE TITLE
Cannot add same sumed columns using different aliases if both queries are the same

### ISSUE THREAD

**Issue by elperepat** [Alexmg86/laravel-sub-query#7]
Hi!

First I want to say thank you for this great package. I've struggled on multiple projects trying to optimize my queries. I always seem to have difficulty with the n+1 problem, both with the summing of columns and the latest for each. So your package will be very useful for me.

However, in one on my project, I need to retrieve the sum of the same relation, but with different aliases. Your code seems to retrieve only the first and disregard the second instance in the withSum array.

For example:
*An Account has many Expenses.
*I want to retrieve all expenses before $period_start and also before $period_end for each account

The following code:
```
$accounts = Account::withSum([
            'expenses:value as expensesStart' => function ($query) use ($period_start) {
            $query->where('date','<',$period_start);
        },
            'expenses:value as expensesEnd' => function ($query) use ($period_end) {
                $query->where('date','<',$period_end);
        }
        ])->get();
```

Generates this query if dates are different:

> select `accounts`.*, (select sum(value) from `expenses` where `accounts`.`id` = `expenses`.`account_id` and `date` < '2021-01-31') as `expensesStart`, (select sum(value) from `expenses` where `accounts`.`id` = `expenses`.`account_id` and `date` < '2021-02-06') as `expensesEnd` from `accounts`

But generates this query if both dates are the same:
> select `accounts`.*, (select sum(value) from `expenses` where `accounts`.`id` = `expenses`.`account_id` and `date` < '2021-02-03') as `expensesStart` from `accounts`

I would like both aliases to be returned, disregarding if both results are the same. Do you think it would be possible?

Thanks again

Patrick

**Comment by Alexmg86** [comment]
Hi Patrick!

Thanks for the comment, it's nice that the package was useful not only to me :)

Indeed, it was my mistake, I added a check for same queries, which was not in Laravel initially, but did not take into account the final names.

I added a small fix in version 1.9.6 (need to update package), so I think everything will be fine now.

`select accounts.*, (select sum(value) from expenses where accounts.id = expenses.account_id and date < '2021-02-03') as expensesStart, (select sum(value) from expenses where accounts.id = expenses.account_id and date < '2021-02-03') as expensesEnd from accounts`
